err_stash.py

```diff
@@ -104,7 +104,7 @@ def fetch_branches(
                 # return [branch for branch in list(repo.get_branches()) if branch_name in branch.name]
                 # slows down the merge plans creation in ~5 to ~10 extra seconds.
                 return []
-            else:
+            else:  # pragma: no cover
```

Comment:
> Wait, why? Feels like this should be covered... 🤔

Since this `else` is only raising exceptions that come from `PyGithub` I thought it wouldn't be necessary to cover, but it doesn't seem difficult to cover (by mocking `PyGithub` and then raising some exception).